app/models/services/base-service.ts

```diff
@@ -321,6 +321,7 @@ export default abstract class BaseService<ServiceParamType> {
       message: message,
       collectionIds: this.operation.collectionIds,
       ignoreErrors: this.operation.ignoreErrors,
+      destination_url: this.operation.destinationUrl,
```

Comment:
I'm trying to figure out the right logic for this, but essentially if the `operation.stagingLocation` is set to something other than the expected default I think we would want to set the destination_url on the request. I think we need to know if the destination_url is external for later tickets, so I think we do want to differentiate between external or our own default location.

Reply:
Right. We can talk about this in the technical meeting today.